src/containers/WebComponentLoader.jsx

```diff
@@ -17,6 +17,7 @@ import ToastCloseButton from "../utils/ToastCloseButton";
 
 import internalStyles from "../assets/stylesheets/InternalStyles.scss";
 import externalStyles from "../assets/stylesheets/ExternalStyles.scss";
+import webComponentStyles from "../assets/stylesheets/WebComponent.scss";
```

Comment:
Can this not be added to the internal styles?

Reply:
Merged it one file, thank you.